tests/std/tests/P2474R2_views_repeat_death/test.cpp

```diff
@@ -81,6 +81,51 @@ void test_iter_add_neg_overflow() {
     it += -1; // integer overflow
 }
 
+template <class I>
+using iota_diff_t = ranges::range_difference_t<ranges::iota_view<I>>;
+
+template <class U>
+constexpr auto positive_huge_diff = iota_diff_t<U>{iota_diff_t<U>{static_cast<U>(-1)} + 1};
+
+template <class U>
+constexpr auto negative_huge_diff = iota_diff_t<U>{-iota_diff_t<U>{static_cast<U>(-1)} - 1};

```

Comment:
Style nitpick: These should use direct-init of `Type var{init}` instead of `auto var = Type{init}`, where `auto` simply obscures what's happening.